ducktape/tests/runner_client.py

```diff
@@ -175,7 +175,8 @@ def _do_run(self, num_runs):
             data = self.run_test()
 
             test_status = PASS
-            self.log(logging.INFO, "{} TEST".format(test_status.to_json()))
+            # test_status.to_json() simply prints PASS - str(self).upper() - so might as well just print PASS directly
+            self.log(logging.INFO, "PASS")
```

Comment:
maybe this print should be moved, as it should print flaky for a flaky test not pass.